docs/content/dagster-cloud/account/setting-up-alerts.mdx

```diff
@@ -5,3 +5,226 @@ noindex: True
+
+       You can then configure the alert policy to message this channel. **Note**: Only messaging one channel per alert policy is currently supported:
+
+       <Image
+       alt="Slack alert configured to alert the sales-notifications channel"
+       src="/images/dagster-cloud/alerts/cloud-slack-alert-creation.png"
+       width={1580}
+       height={511}
+       />
+
+     - **Email** - Email alerts can be sent to one or more recipients. For example:
+
+       <Image
+       alt="Email alert configured to alert two recipients"
+       src="/images/dagster-cloud/alerts/cloud-email-alert-creation.png"
+       width={1571}
+       height={514}
+       />
+
+6. When finished, click **Save policy**.
+
+### Editing alert policies
+
+To edit an existing alert policy, click the **Edit** button next to the policy:
+
+<Image
+alt="Highlighted Edit button next to an alert policy in Dagster Cloud"
+src="/images/dagster-cloud/alerts/edit-alert-policy.png"
+width={1349}
+height={153}
+/>
+
+### Enabling and disabling alert policies
+
+To enable or disable an alert, use the toggle on the left side of the alert policy:
+
+<Image
+alt="Enabling and disabling an alert policy in Dagster Cloud"
+src="/images/dagster-cloud/alerts/enable-disable-alert-policy.gif"
+width={822}
+height={158}
+/>
+
+### Deleting alert policies
+
+To delete an alert policy, click the **Delete** button next to the policy. When prompted, confirm the deletion.
+
+<Image
+alt="Highlighted Delete button next to an alert policy in Dagster Cloud"
+src="/images/dagster-cloud/alerts/delete-alert-policy.png"
+width={1349}
+height={153}
+/>
+
+---
+
+## Managing alert policies with the dagster-cloud CLI
+
+With the [`dagster-cloud` CLI](/dagster-cloud/developing-testing/dagster-cloud-cli), you can:
+
+- [Set an instance deployment's policies](#setting-an-instance-deployments-policies)
+- [View an instance deployment's policies](#viewing-an-instance-deployments-policies)
+- [Configure a Slack alert policy](#configuring-a-slack-alert-policy)
+- [Configure an email alert policy](#configuring-an-email-alert-policy)
+
+### Setting an instance deployment's policies
+
+A list of alert policies can be defined in a single YAML file. After declaring your policies, set them for your instance deployment using the following command:
+
+```bash
+dagster-cloud deployment alert-policies sync -a <ALERT_POLICIES_PATH>
+```
+
+### Viewing an instance deployment's policies
+
+List the policies currently configured on your instance deployment by running:
+
+```bash
+dagster-cloud deployment alert-policies list
+```
+
+### Configuring a Slack alert policy
+
+In this example, we'll configure a Slack notification to trigger whenever a run of a job succeeds or fails. This job, named `sales_job`, has a tag of `{"team", "sales"}`:
```

Comment:
{"team": "sales"} or maybe do it in text to make it clear that it's key-value? "has the `team` tag set to `sales`"

Reply:
I think I just flubbed the formatting here, to be honest. Thanks for calling this out.